lib/openapi_client_generator.rb

```diff
@@ -338,14 +348,20 @@ def priority
 
   class API
     def self.at(definition, parameterizer: OpenAPIClientGenerator::Endpoint::PositionalParameterizer)
+      overrides_path = "lib/openapi/overrides.rb"
+      source = RuboCop::ProcessedSource.new(File.read(overrides_path), 2.3, overrides_path)
```

Comment:
We're using Ruby version `2.3` since Rubocop gets bundled at `0.36.0`. I tried forcing it to be higher and it broke some dependencies of other gems so I wasn't sure what we wanted to do.

Reply:
> I tried forcing it to be higher and it broke some dependencies of other gems so I wasn't sure what we wanted to do.

We can try and take a look at this today.